### ISSUE TITLE
Guild reward can be duped

### ISSUE THREAD

**Issue by zenandreas** [guilds-plugin/Guilds#339]
Hello,

Upon finishg/winning a guild vs guild event, a player receives the reward, as many times he typed /clan war accept or /clan war join (not 100% which one)

So if a player spams /clan war join 10 times, if he wins the clan war, he gets the reward 10x

**Second bug:**

Sometimes, after i win the guild vs guild, i'm not getting teleported out of the map and doesnt let me type anything. I need to relog on the server but im still in the event map.

**Comment by darbyjack** [comment]
Aha! Thanks for reporting this! I see right away where that first bug is. As far as the second bug, is there anything else you can tell me on how to reproduce this? I tried it a couple times and have not gotten it to happen.

Can you try reproducing it with just Guilds and the dependencies it needs? It might be another plugin interfering.

**Comment by wolfebersahd** [comment]
Check for any errors in console when the guild war ends too, that could be useful.

**Comment by zenandreas** [comment]
https://pastebin.com/f3PuDid2

**Comment by wolfebersahd** [comment]
This is familiar, are you amusing any non-vanilla weapons?

**Comment by zenandreas** [comment]
Yes and no. The player in the situation was killed by a mythic mob monster, but no custom items were used in this situation. He was afk and killed by a mob eventually. I will continue testing this because i think it happens in other cases as well. Ultimately i will have to remove the monsters if it perisists.

**Comment by wolfebersahd** [comment]
I had an issue with non-vanilla weapons.

Doe the mythicmob apply a non-vanilla damage type? That's most likely the culprit.

I have already spoken very briefly about and alternate way to handle deaths during guild wars.

Temporarily removing the mob might be a good solution until this issue is resolved.

**Comment by darbyjack** [comment]
Is this reproducible on the latest version?
